codec-dns/src/main/java/io/netty/handler/codec/dns/DohProviders.java

```diff
@@ -0,0 +1,80 @@
+package io.netty.handler.codec.dns;
+
+public class DohProviders {
+
+    public static class DohProvider {
```

Comment:
Is this really necessary? One can use `InetSocketAddress` and move GET/POST flag to encoder; then DohProviders can be removed altogether.

Reply:
I removed DohProviders and DohProvider based on your suggestion.